packages/react/src/Testimonial/Testimonial.module.css

```diff
@@ -2,11 +2,13 @@
   display: grid;
   gap: var(--base-size-16);
   font-family: var(--brand-fontStack-sansSerif);
+  font-size: var(--brand-Testimonial-quoteSize-default);
   color: var(--brand-color-text-default);
 }
 
 .Testimonial::before {
   content: '“';
+  color: var(--testimonial-accent-color);
```

Comment:
This doesn't appear to be assigned to a value, but should be customisable 👍 

Could you hoist this into a colors.json design token source file please?

Reply:
I'm wondering if it makes more sense to assign a default by providing a default value to the `quoteMarkFill` prop. Let me know which you prefer.

Reply:
> I'm wondering if it makes more sense to assign a default by providing a default value to the quoteMarkFill prop

We usually set all default values and design logic variables in the primitive layer first.

Also fine if you want to set it in the React layer for now, as it can always be changed later 👍